src/alert/alert.style.tsx

```diff
@@ -1,85 +1,145 @@
                 break;
             default:
-                backgroundColor = Color.Validation.Green.Background(props);
-                borderColor = Color.Validation.Green.Border(props);
+                backgroundColor = Color.Validation.Orange.Background(props);
+                borderColor = Color.Validation.Orange.Border(props);
+                iconColor = Color.Validation.Orange.Icon(props);
                 break;
         }
 
-        return `
-			background: ${backgroundColor};
-			border-left: 2pt solid ${borderColor};
-		`;
+        return css`
+            background: ${backgroundColor};
+            border-left: 2pt solid ${borderColor};
+
+            svg {
+                color: ${iconColor};
+                width: ${iconSize};
+                height: ${iconSize};
+                margin-right: 0.5rem;
+            }
+        `;
     }}
 
-	p {
-        ${TextStyleHelper.getTextStyle("BodySmall", "regular")}
+    p {
         margin: 0;
+        ${(props) => {
+            if (props.$size === "small")
+                return css`
+                    ${TextStyleHelper.getTextStyle("H6", "regular")}
 
-        strong {
-            ${TextStyleHelper.getFontFamily("BodySmall", "semibold")}
-        }
+                    strong {
+                        ${TextStyleHelper.getTextStyle("H6", "semibold")}
+                    }
+
+                    a {
+                        ${TextStyleHelper.getTextStyle("H6", "semibold")}
+                    }
+                `;
+            else {
+                return css`
+                    ${TextStyleHelper.getTextStyle("H5", "regular")}
+
+                    strong {
+                        ${TextStyleHelper.getTextStyle("H5", "semibold")}
+                    }
+
+                    a {
+                        ${TextStyleHelper.getTextStyle("H5", "semibold")}
+                    }
+                `;
+            }
+        }}
 
         a {
             color: ${Color.Primary};
             text-decoration: none;
-
-            :hover,
-            :active,
-            :focus {
-                color: ${Color.Secondary};
-            }
         }
     }
 `;
 
-export const ActionLinkText = styled(Text.Hyperlink.Small)`
-    margin-bottom: 0;
-    margin-top: 0.5rem;
+export const ActionLinkText = styled(Text.Hyperlink.Small)<StyleProps>`
+    ${(props) => {
+        if (props.$size === "small")
+            return css`
+                ${TextStyleHelper.getTextStyle("H6", "semibold")}
+                margin-top: 0.25rem;
+            `;
+        else {
+            return css`
+                ${TextStyleHelper.getTextStyle("H5", "semibold")}
+                margin-top: 0.5rem;
+            `;
+        }
+    }}
     display: flex;
+    flex-direction: row;
     align-items: center;
+    color: ${Color.Primary};
+
+    svg {
+        display: flex;
+        height: 1rem;
+        width: 1rem;
+        margin-left: 0.25rem;
+        margin-right: 0;
```

Comment:
when the above comment for the `svg` is resolved, `margin-right` will not be needed